community-membership.md

```diff
@@ -113,26 +113,26 @@ only be doing issues triage and have no approval rights.
 ### Requirements
 
 The following apply to the part of the codebase for which one would be an
-approver in the `CODEOWNER` files.
+approver in the `CODEOWNERS` files.
 
-- Reviewer of the codebase for at least 3 months
-- Reviewer for at least 10 substantial PRs to the codebase
-- Reviewed or merged at least 30 PRs to the codebase
+- Reviewer of the codebase for at least 1 month
+- Reviewer for or contributor of at least 10 substantial PRs to the codebase
+- Reviewed or contributed at least 20 PRs to the codebase
 - Nominated by a maintainer
```

Comment:
do we want to quantify what that counts as? or say that approver status may be granted without CODEOWNER, but that CODEOWNER requires triaging/docs?

Reply:
today every approver needs to be added to the team foo-approvers and to the `CODEOWNERS` file. It is nice to have a `CODEOWNERS` file as a source of truth as it is controlled by pull requests.

Having a triager role makes the matter more complicated. Person needs to be in foo-approvers team, but not necessarily in `CODEOWNERS` file. Perhaps members like this may be added into the commented section.

As for quantification of the triager contributions - maybe the same time line of 1 month helping triage issues or helped with 10 substential issues resolution may be a good qualification criteria.

Reply:
Deferring the triager discussion for later.